OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -34,12 +30,8 @@
 
 import roboguice.inject.InjectView;
 
-
-/**
- *
- */
 public class CourseUnitNavigationActivity extends CourseBaseActivity implements CourseUnitVideoFragment.HasComponent {
-
+    private static final byte UNITS_TO_RELOAD = 3;
```

Comment:
3 means at one time, 7 fragments/pages will be loaded. Earlier we were loading just one page initially and 3 later, its significant jump, lets restrict to 2 (5 pages load at one time) for now.

Reply:
done!